python/py-httpbin/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-httpbin
+version             0.7.0
+revision            0
+
+platforms           darwin
+license             MIT
+maintainers         nomaintainer
+
+description         HTTP Request and Response Service
+long_description    ${description}
+
+homepage            https://github.com/requests/httpbin
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+distname            ${python.rootname}-${version}
+
+checksums           sha256  cbb37790c91575f4f15757f42ad41d9f729eb227d5edbe89e4ec175486db8dfa \
+                    rmd160  22b1bb0151212aa71aaa8cb8fb1aec7e1e9c0b5a \
+                    size    92613
+
+python.versions     37
+
+if {${name} ne ${subport}} {
+    depends_build-append \
+                    port:py${python.version}-setuptools
+
+    depends_lib-append \
+                    port:py${python.version}-brotlipy \
+                    port:py${python.version}-decorator \
+                    port:py${python.version}-flask \
+                    port:py${python.version}-itsdangerous \
+                    port:py${python.version}-markupsafe \
+                    port:py${python.version}-raven \
+                    port:py${python.version}-six \
+                    port:py${python.version}-werkzeug
+
+    test.run        yes
```

Comment:
there are two tests failing here, but that seems to be an -already reported to upstream- [issue](https://github.com/postmanlabs/httpbin/issues/554) and not something with how we package it.

Reply:
Thanks a lot for the link. It was too late to keep exploring.

There is another error in trace mode:
```
  File "/opt/local/Library/Frameworks/Python.framework/Versions/3.7/lib/python3.7/site-packages/raven/conf/defaults.py", line 48, in <module>
    CA_BUNDLE = certifi.where()
AttributeError: module 'certifi' has no attribute 'where'
```